backend/validators/task.js

```diff
@@ -0,0 +1,11 @@
+const { body, param } = require('express-validator');
+
+const store = [body('name').trim().notEmpty().withMessage('Task name is required')];
+const update = [
+    ...store,
+    ...destroy,
+    body('completed').trim().notEmpty().withMessage('Task completed flag is required')
+];
+const destroy = [param('id').trim().notEmpty().withMessage('Task id is required')];
```

Comment:
not sure if needed, I think wen `id` will be empty then it will be different route

Reply:
You re right. If id query parameter will be empty then request will return Cannot DELETE /api/tasks/. Done